### PR TITLE
Add convenience API Sorts.make taking quality and universe

### PR DESCRIPTION
**Pull request by SkySkimmer**

### CODE REVIEW

**Review by ppedrot — APPROVED** [review]

**Comment by ppedrot** [comment]
@coqbot run full ci

File: kernel/sorts.mli [hunk 1]
```diff
@@ -142,6 +142,7 @@ val set  : t
 val prop : t
 val type1  : t
 val qsort : QVar.t -> Univ.Universe.t -> t
+val make : Quality.t -> Univ.Universe.t -> t
```
Comment (herbelin) [inline kernel/sorts.mli:145]: Maybe adding a bit of documentation explaining the role of `qsort` and `make`?

Comment (ppedrot) [reply kernel/sorts.mli:145]: For this kind of trivial wrapper I don't think this is required.

**Comment by ppedrot** [comment]
@coqbot merge now

**Pull request merged**